blog/_posts/2017-05-19-introducing-the-new-goodtables-library-and-goodtablesio.md

```diff
@@ -2,102 +2,105 @@
+## The goodtables library
 
-Installation of goodtables v1 is as easy as  
-**`pip install goodtables --pre`**
+[goodtables](https://github.com/frictionlessdata/goodtables-py/) is a Python library that allows users to inspect tabular data, checking it for both structural and schematic errors, and giving pointers on plausible error fixes, before users draw analyses on the data using other tools. At its most basic level, goodtables highlights general errors in tabular files that would otherwise prevent loading or parsing.
 
-goodtables v1  supports the **_table_**, **_tables_** and **_datapackage_** presets by default. For the Examples below, we will use datasets [from this folder](https://github.com/frictionlessdata/goodtables-py/tree/master/data).  
-The **_table_** preset allows you to inspect a single tabular file. Programmatically, you can also inspect multiple tabular files in parallel using the tables preset.  
-  _Example:_  
-  **`goodtables --json table valid.csv`** 
-  returns the JSON Table Schema (JTS) for the specified file, specifies error count and states source and validity of the data file among other things.
+Since [the release of goodtables v0.7 in early 2015](http://okfnlabs.org/blog/2015/02/20/introducing-goodtables.html), the codebase has evolved, allowing for additional use cases while working with tabular data.  Without cutting back on functionality, goodtables v1 has been simplified, and the focus is now on extensible data validation.
 
-Data Packages are a format for coalescing data in one ‘container’ before shipping it for use by different people and with different tools. In goodtables, the datapackage preset allows you to run checks on datasets aggregated in one container. Working with data package Examples [from this repository](https://github.com/frictionlessdata/goodtables-py/tree/master/data/datapackages):  
-  _Example:_  
-  **`goodtables datapackage datapackage.json`**  
-  allows a user to check a data package’s schema, table by table, and gives a detailed report on errors, row count, headers, and validity or lack thereof of a data package.
+## Using goodtables
 
-In addition to general structure and schema checks on tabular files available in v0.7, the goodtables library now allows users to define custom data presets and run custom checks on tabular files. So what is the difference?
+goodtables is still in alpha, so we need to pass the pre-release flag (`--pre`) to `pip` to install.  With that, installation of goodtables v1 is as easy as `pip install goodtables --pre`.
 
-While basic schema checks inspect data against this data quality spec , **_custom data checks_** give users leeway to specify acceptable values against data fields so that any values outside of the defined rules are flagged as errors.
+goodtables v1 supports two presets by default: **_table_**  and **_datapackage_**.  The `table` preset allows you to inspect a single tabular file.
```

Comment:
There is a new preset (`nested`) that basically allows you to pass a list of tables or datapackages, although this can't be used from the command line AFAICT so probably not worth mentioning.